.github/workflows/e2e-node.yml

```
@@ -15,11 +15,17 @@ jobs:
     environment:
       name: ${{ matrix.environment-name }}
     strategy:
+      fail-fast: true
       matrix:
         os: [ubuntu-latest]
-        node-version: [16.x]
-        # NSS does not support static client registration, which we rely on for testing.
-        environment-name: ["ESS Dev-Next"]
+        node-version: ["14.x", "16.x"]
+        environment-name: ["ESS PodSpaces", "ESS PodSpaces Next"]
```

```suggestion
        environment-name: ["ESS PodSpaces"]
```

ESS PodSpaces Next no longer exists. There is only ESS PodSpaces now